### ISSUE TITLE
Improvement in  jcvi.annotation.train augustus

### ISSUE THREAD

**Issue by xuzhougeng** [tanghaibao/jcvi#150]
When I run the follwoing code

```bash
 python -m jcvi.annotation.train augustus \
    --augustus_home=/opt/biosoft/maker/exe/augustus \
    New_Species \
    transcripts.gff3 \
    reference.fasta &
```

I came cross with an error 

```bash
cp: cannot stat '/opt/biosoft/maker/exe/augustus/config/species/New_Species ': No such file or directory
```

I think the it was caused by the `new_species.pl` 

```bash
Environment variable AUGUSTUS_CONFIG_PATH not set. at /opt/biosoft/maker/exe/augustus/scripts/new_species.pl line 48
```

To solve this problem, I need add parameter `--AUGUSTUS_CONFIG_PATH=/opt/biosoft/maker/exe/augustus/config/species` to the `new_species.pl` 

So I change some code in jcvi/annotation/train.py

Before 

```python
sh("{0}/scripts/new_species.pl --species={1}".format(mhome, species))
```

After 

```python
config_path = "{0}/config".format(mhome)
sh("{0}/scripts/new_species.pl --species={1} --AUGUSTUS_CONFIG_PATH={2}".format(mhome, species, config_path))
```

And it works successfuly now in my server now.

**Comment by tanghaibao** [comment]
@xuzhougeng 

Thanks for the suggestions. Would you please open a pull request? I'll merge it in.

Haibao

**Comment by xuzhougeng** [comment]
@tanghaibao  Thanks for your advice, I have opened a pull request https://github.com/tanghaibao/jcvi/pull/151#issue-312127594

**Comment by tanghaibao** [comment]
Fixed in 9be6a38.
